openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -372,6 +379,7 @@ def test_patch_account_unknown_user(self, api_client, user):
             "bio", u"<html>Lacrosse-playing superhero 壓是進界推日不復女</html>",
             "z" * 3001, u"Ensure this field has no more than 3000 characters."
         ),
+        ("account_privacy", ALL_USERS_VISIBILITY),
```

Comment:
I believe ALL_USERS_VISIBILITY is the default value. Could you also test the other value (just add another case to the BDD data)?